tests/rov_pressure_test.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+import ms5837
+import time
+
+sensor = ms5837.MS5837_30BA() # Default I2C bus is 1 (Raspberry Pi 3)
+
+# We must initialize the sensor before reading it
+if not sensor.init():
+        print "Sensor could not be initialized"
+        exit(1)
+
+# We have to read values from sensor to update pressure and temperature
+if not sensor.read():
+    print "Sensor read failed!"
+    exit(1)
+
+#print("Pressure: %.2f mbar") % (sensor.pressure())
+
+#print("Temperature: %.2f C") % (sensor.temperature(ms5837.UNITS_Centigrade))
+
+#time.sleep(2)
+
+print("Time \tPressure (mbar) \tTemperature (C)")
+
+# Spew readings
+while True:
+        if sensor.read():
+            print("%s \t%0.1f \t%0.2f") % (time.strftime("%H:%M:%S", time.localtime()) + '.%d' % (time.time() % 1 * 1000),
```

Comment:
So the tests/ folder is primarily for Unit Tests. This seems more like just a start script. This is good to have. I would put this exact code at the bottom of the pressure code under a `if __name__ == '__main__':` so we can make it run by running the ms5837 file. And make another for the tests/ folder that just checks whether a reasonable value is generated from the sensor and whether it changes. I think that would be a good enough Unit Test that the sensor code is working (for automated testing).